libs/go/sia/host/provider/provider.go

```diff
@@ -64,6 +64,9 @@ type Provider interface {
 	// GetSuffix returns the suffix for the current provider
 	GetSuffix() string
 
+	// GetSuffixes returns a list of suffixes for the current provider
+	GetSuffixes() []string
```

Comment:
why keep the GetSuffix() api? since this is an internal interface for now and only used within the server, we can just update all the provider implementations to use GetSuffixes() instead of maintaining a deprecated api